<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Amplifier</class>
 <widget class="QMainWindow" name="Amplifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amplifier</string>
  </property>
  <property name="accessibleName">
   <string>Amplifier's settings window</string>
  </property>
  <property name="accessibleDescription">
   <string>Allows you to set amplifier's settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&amp;Amplifier:</string>
        </property>
        <property name="buddy">
         <cstring>comboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="accessibleName">
         <string>Amplifier</string>
        </property>
        <property name="accessibleDescription">
         <string>Allows you to choose amplifier to emulate</string>
        </property>
        <property name="maxVisibleItems">
         <number>19</number>
        </property>
        <item>
         <property name="text">
          <string>Fender '57 Deluxe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fender '59 Bassman</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fender '57 Champ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fender '65 Deluxe Reverb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fender '65 Princeton</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fender '65 Twin Reverb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fender Super-Sonic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>British 60's</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>British 70's</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>British 80's</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>American 90's</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Metal 2000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rumble</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bassman TV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bassman 300</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KGB 800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rockin' Peg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SWR Redhead</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monster</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="advancedButton">
        <property name="accessibleName">
         <string>Advanced button</string>
        </property>
        <property name="accessibleDescription">
         <string>Opens window winth advanced settings</string>
        </property>
        <property name="text">
         <string>A&amp;dvanced</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accessibleName">
         <string>Set button</string>
        </property>
        <property name="accessibleDescription">
         <string>Sends changes to the amplifier</string>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QDial" name="dial">
          <property name="accessibleName">
           <string>Gain dial</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to choose gain level</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="notchTarget">
           <double>12.750000000000000</double>
          </property>
          <property name="notchesVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="accessibleName">
           <string>Gain box</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to precisely enter the level of gain</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;Gain</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBox</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QDial" name="dial_2">
          <property name="accessibleName">
           <string>Volume dial</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to choose volume level</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="notchTarget">
           <double>12.750000000000000</double>
          </property>
          <property name="notchesVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_2">
          <property name="accessibleName">
           <string>Volume box</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to precisely enter the level of volume</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&amp;Volume</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBox_2</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QDial" name="dial_3">
          <property name="accessibleName">
           <string>Treble dial</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to choose treble level</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="notchTarget">
           <double>12.750000000000000</double>
          </property>
          <property name="notchesVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_3">
          <property name="accessibleName">
           <string>Treble box</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to precisely enter the level of treble</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&amp;Treble</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBox_3</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QDial" name="dial_4">
          <property name="accessibleName">
           <string>Middle tones dial</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to choose middle tones level</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="notchTarget">
           <double>12.750000000000000</double>
          </property>
          <property name="notchesVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_4">
          <property name="accessibleName">
           <string>Middle tones box</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to precisely enter the level of middle tones</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&amp;Middle</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBox_4</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QDial" name="dial_5">
          <property name="accessibleName">
           <string>Bass dial</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to choose the bass level</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="notchTarget">
           <double>12.750000000000000</double>
          </property>
          <property name="notchesVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_5">
          <property name="accessibleName">
           <string>Bass box</string>
          </property>
          <property name="accessibleDescription">
           <string>Allows you to precisely enter the level of bass</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&amp;Bass</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBox_5</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
